swap/tests/harness/mod.rs

```diff
@@ -58,12 +57,16 @@ struct BobParams {
 }
 
 impl BobParams {
-    pub async fn builder(&self, event_loop_handle: bob::EventLoopHandle) -> Result<bob::Builder> {
+    pub async fn builder(
+        &self,
+        event_loop_handle: bob::EventLoopHandle,
+        swap_id: Uuid,
+    ) -> Result<bob::Builder> {
         let receive_address = self.monero_wallet.get_main_address();
 
         Ok(bob::Builder::new(
```

Comment:
I think we should get rid of this builder on the next opportunity, I don't see any purpose in that abstraction anymore :)